.travis.yml

```diff
@@ -41,13 +41,13 @@ addons:
             - libatlas-base-dev
             - liblapack-dev
             - gfortran
-            - cmake
 
 before_install:
     - sudo apt-get -qq update
     - if [[ "$TOXENV" != "py35-nodeps" ]]; then sudo apt-get install graphviz; fi
 
 install:
+    - pip install --upgrade cmake
```

Comment:
Correct! Previously XGBoost couldn't build because CMake 3.13 is required, but apt installed 3.12.4 (https://travis-ci.org/github/TeamHG-Memex/eli5/jobs/704783435). Using pip seemed like the easiest way to install the newest version.

Reply:
Makes sense, thanks 👍